zrml/swaps/src/benchmarks.rs

```diff
@@ -248,33 +251,76 @@ benchmarks! {
         });
     }: admin_clean_up_pool(RawOrigin::Root, market_id, OutcomeReport::Scalar(33))
 
-    end_subsidy_phase {
-        // Total assets
-        let a in (T::MinAssets::get().into())..T::MaxAssets::get().into();
-        // Total subsidy providers
-        let b in 0..10;
+    // This is a worst-case benchmark for arbitraging a number of pools.
+    apply_to_cached_pools_execute_arbitrage {
+        let a in 0..63; // The number of cached pools.
```

Comment:
Is there a reason why you use a numeric constant instead of the `Config` value?

Reply:
Which `Config` value would you use here? I never defined a maximum number of pools per block or anything like that?

Reply:
My bad I got confused there, we have no `Config` value for that :man_facepalming:

Reply:
No worries, buddy